test/unit/org/apache/cassandra/index/sai/cql/IndexCompressionTest.java

```diff
@@ -75,25 +77,14 @@ public void testKeyCompression()
     }
 
     @Test
-    public void testKeyCompressionIsCommonForAllIndexes()
+    public void testKeyCompressionMustMatchOnAllIndexes()
```

Comment:
Nit: The comment below needs an update. Also, this latter change leaves some unused imports.

Reply:
Fixed